modules/ipcCommunicator.js

```diff
@@ -95,6 +96,26 @@ ipc.on('backendAction_importPresaleFile', function(e, path, pw) {
     var error = false;
 
     // start import process
+    fs.readFile(path, 'utf8', function (e, data) {
+        if(!e) {
+            try {
+                var wallet = JSON.parse(data); 
+            } catch (err) {
+                log.error("Wallet import: Cannot read file");
+                log.error(err);
+            }
+            if(data.indexOf('"ethaddr"') !== -1) {  // Presale wallet
+                console.info("presale");
```

Comment:
What is the best way to execute all the following commands? 
Nesting them here? Creating new functions(where)?